tests/system/general/test_45_FilterSolutions/test_filter/test_attributes.py

```diff
@@ -44,7 +45,8 @@
 @pytest.mark.skipif(config["desktopVersion"] < "2025.1", reason="Skipped on versions earlier than 2025.1")
 class TestClass:
     def test_filter_type(self):
-        design = ansys.aedt.core.FilterSolutions(implementation_type=FilterImplementation.LUMPED)
+        design = ansys.aedt.core.filtersolutions.LumpDesign(config["desktopVersion"])
```

Comment:
Switching to a fixture would mean future changes like this would be a one-line change in one place. Would be a good idea to apply that in this PR I think, since we are already editing all these lines.